Swift/Replicator.swift

```diff
@@ -175,6 +175,35 @@ public final class Replicator {
         _impl.removeChangeListener(with: token._impl)
     }
     
+    /// Gets a set of document Ids, who have revisions pending push. This API is a snapshot and results
+    /// may change between the time the call was made and the time the call returns.
+    ///
+    /// - Returns: A  set of document Ids, each of which has one or more pending revisions
+    public func pendingDocumentIds() throws -> Set<String> {
+        var error: NSError?
+        let result = _impl.pendingDocumentIds(&error)
```

Comment:
The objective-C -pendingDocumentIds should return null if there is an error occurred. Then this place we an let the objective-c thrown in swift so we don't need to check the error.

Reply:
that makes sense